Makefile

```diff
@@ -28,9 +28,18 @@ build:
 	cargo build --locked -p tuftool
 	cargo test --locked
 
-# checks tough tests with and without the http feature. http testing requires docker.
+
+# installs noxious-server
+# We currently build from a forked version, until such a point that the following are resolved:
+# https://github.com/oguzbilgener/noxious/issues/13
+# https://github.com/oguzbilgener/noxious/pull/14
+.PHONY: noxious
+noxious:
+	cargo install --locked --git https://github.com/cbgbt/noxious.git --tag v1.0.5
```

Comment:
Can we add a tracking issue to make sure we don't forget about this fork.

Reply:
Yes, I opened one here: https://github.com/awslabs/tough/issues/671